src/js/shoppingCart.mjs

```diff
@@ -28,5 +28,16 @@ export default async function cartList(){
   } else {
     const parentEl = document.querySelector('.product-list');
     renderListWithTemplate(cartItemTemplate, parentEl, cartItems);
+
+    // Delete 'hide' class in div 'cart-footer'
+    document.querySelector('div.cart-footer').classList.remove('hide');
```

Comment:
why do we need to put the `hide` class on there in the first place if we remove it when we render it?

Reply:
The total should be hidden by default, but will be displayed if there are items in the cart.
Or that's what I understood